docs/source/development-guide/docker.rst

```diff
@@ -0,0 +1,28 @@
+Docker Workflow
+----------------
+
+This page describes how to build and run a Docker Image Locally and in AWS.
+
+Building and Running a Docker Image Locally
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To build the image run the following command from the Dockerfile directory. You might add -t option to tag your image
+and --rm to remove intermediate containers after the build is done.
+
+`docker build -t my-image --rm .`
+
+Now we can run our image. Example below runs the image and executes the ois-ingest command with the str option.
+
+`docker run --rm -it my-image:latest ois-ingest --ccsds str`
```

Comment:
```suggestion
To build the image run the following command from the Dockerfile directory. You might add -t option to tag your image
and --rm to remove intermediate containers after the build is done.::

    docker build -t my-image --rm .

Now we can run our image. Example below runs the image and executes the ois-ingest command with the str option::

    docker run --rm -it my-image:latest ois-ingest --ccsds str
```

I think doing this will render these as code blocks in the sphinx docs